testing/python/fixture.py

```diff
@@ -2547,6 +2548,44 @@ def test_foo(fix):
             '*test_foo*alpha*',
             '*test_foo*beta*'])
 
+    @pytest.mark.issue920
+    def test_deterministic_fixture_collection(self, testdir):
```

Comment:
Please use the `monkeypatch` fixture instead of importing and creating your own `MonkeyPatch` instance.